api/models.py

```diff
@@ -110,10 +110,10 @@ def save(self, *args, **kwargs):
         are provided and an existing status is given
```

Comment:
and this - line [109](https://github.com/AndelaOSP/art-backend/pull/57/files#diff-9dba586f38463d27c2d7fdadad3eb040R109)

Reply:
Done.